eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -470,6 +470,10 @@ class Router(val nodeParams: NodeParams, watcher: ActorRef, initialized: Option[
       sender ! updates
       stay
 
+    case Event('updatesMap, d) =>
```

Comment:
This is the same as 'channelsMap, probably don't need to add it, just call 'channelsMap instead?

Reply:
Yes they're the same and 'updatesMap can go, i removed it in fe70ae2.